packages/db/src/migrations/0020_add_article.ts

```diff
@@ -0,0 +1,29 @@
+import { type Kysely, sql } from "kysely"
+import { createTableWithDefaults } from "../utils"
+
+export async function up(db: Kysely<any>) {
+  await createTableWithDefaults("articles", { id: true, createdAt: true, updatedAt: true }, db.schema)
+    .addColumn("title", "text", (col) => col.notNull())
+    .addColumn("author", "text", (col) => col.notNull())
+    .addColumn("photographer", "text", (col) => col.notNull())
+    .addColumn("image_url", "text", (col) => col.notNull())
```

Comment:
Should it be mandatory to upload an image for the article?

Reply:
I think it's reasonable to say so, I don't think there has been an article on OW without an image for as long as I've been a student

Reply:
Fair enough!